src/events/deposits.controller.ts

```diff
@@ -68,4 +71,13 @@ export class DepositsController {
     await this.depositsUpsert.bulkUpsert(data.operations);
     res.sendStatus(HttpStatus.ACCEPTED);
   }
+
+  @ApiExcludeEndpoint()
+  @Post('fix_mismatches')
```

Comment:
What do you think about `refresh`, for consistency?

Reply:
I like it, I'll change the endpoint and rename the associated methods